java/client/src/main/java/glide/api/models/commands/ScriptOptions.java

```diff
@@ -0,0 +1,24 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.models.Script;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Singular;
+
+/**
+ * Optional arguments for {@link glide.api.commands.GenericBaseCommands#invokeScript(Script,
+ * ScriptOptions)} command.
+ *
+ * @see <a href="https://redis.io/commands/evalsha/">redis.io</a>
```

Comment:
Don't you need `<a href="https://redis.io/commands/script-load/">SCRIPT LOAD</a>` here too?

Reply:
The LOAD script doesn't take the options.  It's the EVAL call that takes the options.